includes/data-port/models/class-sensei-data-port-lesson-schema.php

```diff
@@ -131,7 +132,28 @@ public function get_schema() {
 	}
 
 	/**
-	 * Get question post type.
+	 * Validate passmark.
+	 *
+	 * @access private
+	 *
+	 * @param string                     $field Field name.
+	 * @param Sensei_Import_Lesson_Model $model Lesson model.
+	 *
+	 * @return boolean Is valid.
+	 */
+	public function validate_passmark( $field, Sensei_Import_Lesson_Model $model ) {
```

Comment:
If we have many validations, we could create a validation lib and it could become a range validator in this lib.